packages/amesos/CMakeLists.txt

```diff
@@ -9,10 +9,11 @@ TRIBITS_PACKAGE(Amesos)
 #
 # B) Set up package-specific options
 #
-
-# if using SuperLUDist, must also link in ParMETIS for some reason
-IF(${PACKAGE_NAME}_ENABLE_SuperLUDist AND NOT ${PACKAGE_NAME}_ENABLE_ParMETIS)
-  MESSAGE(FATAL_ERROR "The Amesos support for the SuperLUIDist TPL requires the ParMETIS TPL.  Either disable Amesos SuperLUDist support or enable the ParMETIS TPL.")
+# One can now configure SuperLUDist without ParMETIS
+if (NOT TPL_ENABLE_SuperLUDist_Without_ParMETIS)
```

Comment:
Since this option is really specific to Amesos, this should be called something like `Amesos_Allow_SuperLUDist_Without_ParMETIS` and it should be defined as a documented advanced cache var like:

```
ADVANCED_SET(Amesos_Allow_SuperLUDist_Without_ParMETIS OFF
  CACHE BOOL "Allow Amesos to use the SuperLUDist TPL without the enable of the ParMETIS TPL")
```

and then replace `TPL_ENABLE_SuperLUDist_Without_ParMETIS` with `Amesos_Allow_SuperLUDist_Without_ParMETIS` in this if statement.  (In which case you should loose the above comment.)

The name  `TPL_ENABLE_SuperLUDist_Without_ParMETIS` is too close to `TPL_ENABLE_SuperLUDist` which makes it seem this this is a TriBITS-supported var which it is not.  It is specific to Amesos and therefore should be prefixed with `Amesos`.